ods_ci/build/run.sh

```diff
@@ -1,28 +1,32 @@
 #!/bin/bash
+set -e
```

Comment:
awesome, +100

```suggestion
set -Eeo pipefail
```

This is the recommendation from https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/, leaving out `-u` because that needs more work